tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -2550,7 +2559,29 @@ int tracepoint__sched__sched_switch(struct bpf_raw_tracepoint_args *ctx)
 
     return events_perf_submit(&data, SCHED_SWITCH, 0);
 }
+SEC("kprobe/filldir64")
+int BPF_KPROBE(trace_filldir64)
+{
+    event_data_t data = {};
+    if (!init_event_data(&data, ctx))
+        return 0;
+    if (!should_trace((&data.context)))
+        return 0;
 
+    char* top = "top";
+    char* ps = "ps";
+    //check's if the called process is top or ps (to list processes)
+    if ((!local_strncmp(top, data.context.comm, 3)) && (!local_strncmp(ps, data.context.comm, 2)))
```

Comment:
We already have the "has_prefix" function - can't it be used instead local_strncmp that you added here?

Reply:
i removed the entire verification- see my last comment